src/tlo/methods/symptommanager.py

```diff
@@ -396,6 +400,11 @@ def change_symptom(self, person_id, symptom_string, add_or_remove, disease_modul
             self.bsh.set(person_id, disease_module.name, columns=sy_columns)
             self._persons_with_newly_onset_symptoms = self._persons_with_newly_onset_symptoms.union(person_id)
 
+            # Update symptom tracker
+            for pid in person_id:
+                for sym in symptom_string:
```

Comment:
Rather than looping through symptoms, might be that doing the union operation of the current set of symptoms with the new set of symptoms is faster